addons/mail/static/src/utils.js

```diff
@@ -56,3 +56,37 @@ export function useAutoScroll(refName) {
         }
     });
 }
+
+export function useHover(refName) {
+    const ref = useRef(refName);
+    const state = useState({ isHover: false });
+    onMounted(() => {
+        ref.el.addEventListener("mouseenter", () => state.isHover = true, true);
+        ref.el.addEventListener("mouseleave", () => state.isHover = false, true);
+    });
+    onWillUnmount(() => {
+        ref.el.removeEventListener("mouseenter", () => state.isHover = true, true);
+        ref.el.removeEventListener("mouseleave", () => state.isHover = false, true);
+    });
```

Comment:
Can have simpler implementation with `useExternalListener`

Reply:
I tried to use this event and in this case, I have to use the `mousemove` event. So I wonder if it is better to use this implementation.
And do you think it is a good idea to make a `useHover` for not only one ref? Like having a list on the ref we want to check and bind the event listeners at once. So we don't have to write when we have `
this.xxHover = useHover("xx");
this.yyHover = useHover("yy");`

Reply:
For now better have as many `useHover` as needed. It's consistent with other component hooks like `useRef()`.